results/index.php

```diff
@@ -138,6 +152,8 @@ function format($d){
 $jitMeterBbox=imageftbbox($FONT_METER_SIZE,0,$FONT_METER,$jit);
 $mbpsBbox=imageftbbox($FONT_MEASURE_SIZE_BIG,0,$FONT_MEASURE,$MBPS_TEXT);
 $msBbox=imageftbbox($FONT_MEASURE_SIZE,0,$FONT_MEASURE,$MS_TEXT);
+$serverBbox=imageftbbox($FONT_SERVER_SIZE,0,$FONT_SERVER,$server);
+$POSITION_X_SERVER=$WIDTH-$serverBbox[4]-4*$SCALE;
```

Comment:
I am not sure if this is the correct position, it is the same as used above but at least for me it looks like it does not fit perfectly.

![download](https://user-images.githubusercontent.com/26308977/77202179-1d074d80-6aee-11ea-8b93-205dd31713cd.png)